site/app/templates/submission/regrade/Discussion.twig

```diff
@@ -10,38 +78,88 @@
         </div>
     </div>
 {% endfor %}
-
-{% if core.getUser().accessGrading() %}
-    {% set reply_message = "Reply" %}
-    {% set reply_placeholder = "Enter your reply here" %}
-{% else %}
-    {% set reply_message = "Request further TA/Instructor action" %}
-    {% set reply_placeholder = "If you believe you require more review, enter a reply here to request further TA/Instructor action..." %}
-{% endif %}
-
-{% if core.getUser().accessGrading() or gradeable.getRegradeStatus() != 0 %}
+{% if action == 'make_request_post' %}
 <div style="padding:20px;">
-    <form method="POST" id="replyTextForm" action="{{ core.buildUrl({'component': 'student',
+    <form method="POST" id="replyTextForm" value="-1" action="{{ core.buildUrl({'component': 'student',
             'action': 'make_request_post',
             'regrade_id': thread_id,
             'gradeable_id': gradeable.getId(),
             'user_id': core.getUser().getId()
         }) }}">
-        <textarea name="replyTextArea" id="replyTextArea" style="resize:none;min-height:100px;width:100%; font-family: inherit;" rows="10" cols="30" placeholder="{{ reply_placeholder }}" id="makeRequestPost" required></textarea>
-        <input type="submit" value="{{ reply_message }}" id="submitPost" class="btn btn-default" style="margin-top: 15px; float: right">
+                <input type="hidden" id="regrade" name="regrade" value=thread_id><input type="hidden" id="g_id" name="g_id" value=gradeable.getId()><input type="hidden" id="u_id" name="u_id" value=core.getUser().getId()><input type="hidden" id="status" name="status" value=-1>
```

Comment:
Couple things: 
- Gotta use `{{ stuff }}` if you want to get values from Twig
- Why not put all of these on their own line? Most of this block can be formatted onto more lines for readability.